tensorboard/webapp/deeplink/hash.ts

```diff
@@ -23,25 +23,28 @@ const TAB = '__tab__';
 
 @Injectable()
 export class HashDeepLinker implements DeepLinkerInterface {
```

Comment:
Can we expand this PR to the point where we are accessing tf_global and tf_storage consistently in the Angular part of the app? Basically get to the point where we can delete TfStorageElement and TfGlobalsElement types. The inconsistency in how we access these types is going to hurt somebody's head in the future.

Alternatively: Perhaps the additional cleanup work can be done in a followup PR is you'd like to keep this PR smaller.

Reply:
I think I have addressed this, but I'm going to let you resolve this to be sure